package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "csharp",
   "publisher": "ms-vscode",
-  "version": "1.15.1",
+  "version": "1.15.2",
```

Comment:
It might be worth have a `1.15.2-beta1` for this change. That way, we can validate it (maybe @filipw can too) and then build the final with just a version number change.